packages/eslint-plugin-rules/README.md

```diff
@@ -0,0 +1,68 @@
+# Eslint Plugin Rules
+
+## About
+
+Place for eslint custom rules for Foreman and plugins
+
+## Usage
+
+### Create an eslintrc file
+```js
+{
+  "plugins": ["@theforeman/rules"],
+  "rules": {
+    "@theforeman/rules/require-ouiaid": "warn",
+  }
+}
+```
+These components would be checked for ouiaId field if none is specified in the eslintrc file:
+```
+  "Alert",
+  "Breadcrumb",
+  "Button",
+  "Card",
+  "Checkbox",
+  "Chip",
+  "ChipGroup",
+  "ContextSelector",
+  "Dropdown",
+  "DropdownItem",
+  "DropdownSeparator",
+  "DropdownToggle",
+  "DropdownToggleCheckbox",
+  "FormSelect",
+  "Menu",
+  "Modal",
+  "ModalContent",
+  "Nav",
+  "NavExpandable",
+  "NavItem",
+  "OptionsMenu",
+  "Pagination",
+  "Radio",
+  "Select",
+  "Switch",
+  "TabButton",
+  "TabContent",
+  "Tabs",
+  "Text",
+  "TextInput",
+  "Title",
+  "Toolbar",
+  "Table",
+  "TableComposable",
+  "Tr"
+```
+
+You can specify what components you want to check against.
+```js
+{
+  "plugins": ["@theforeman/rules"],
+  "rules": {
+    "@theforeman/rules/require-ouiaid": [
+      "warn",
+      "Button",
+      "Table",
```

Comment:
Is the warning level really the first element in the array with all the other components? That seems odd..

Reply:
Yep, that is the [format](https://eslint.org/docs/latest/extend/custom-rules#accessing-options-passed-to-a-rule).